recipes/default.rb

```diff
@@ -84,7 +84,6 @@
       group node['ntp']['conf_group']
       mode '0644'
       source 'ntp.leapseconds'
```

Comment:
I think the correct fix would be to set this to the node attribute and then default the attribute to `'ntp.leapseconds'`

Reply:
Do uou mean something like: `default['ntp']['leapfile_source'] = 'ntp.leadpseconds'` in *attributes/default.rb*?

Reply:
yep, thats it